### ISSUE TITLE
3.0: Mis-alignment of action buttons on customer details screen

### ISSUE THREAD

**Issue by pippinsplugins** [easydigitaldownloads/easy-digital-downloads#6870]
`Edit Profile` and `Switch to User` should be in line: 
![image](https://user-images.githubusercontent.com/1034109/44561353-0f26f600-a71a-11e8-82bd-69325686f151.png)
Note: you need to have the User Switching plugin active to see this.

**Comment by SDavisMedia** [comment]
I've got some thoughts on this. The short of it is I don't think this is 3DD's problem.

* This scenario _also_ requires our internal [EDD Dev Tools plugin](https://github.com/easydigitaldownloads/edd-dev-tools) to be activated. It's in this plugin that we check for User Switching, then add the button to the Customer card (from the Dev Tools plugin, we control whether or not it's a button).
* In EDD 2.X, the `Edit Profile` button reads `Edit Customer`, and it's just a link, not a button. In EDD 3.0, it's changed to a button, which I think is a good enhancement.
* There's no problem until everything comes together in 3DD and the buttons clash. I'm pretty confident that unless there's a drastic style changed to that entire area, having any more than just one button is going to cause a problem.

Considering the above points, I think 3DD core should claim that real estate for itself and nothing else. Let `Edit Profile` remain a button, and stay where it is with no possibility of other buttons/links. In other words, `wontfix`. Might even be a good idea to remove the `edd_after_customer_edit_link` action completely unless someone knows why else it would exist. It was added here: https://github.com/easydigitaldownloads/easy-digital-downloads/issues/5560 (Cc: @cklosowski) 

Then in EDD Dev Tools, we can move the `Switch to User` button somewhere else, most likely turning it into a link. Off the top of my head, I'd say putting it somewhere near the link to the user account/disconnect user. It makes more sense there anyway.

@pippinsplugins @JJJ Thoughts?
